x-pack/plugins/ml/public/application/management/jobs_list/components/jobs_list_page/jobs_list_page.tsx

```diff
@@ -242,17 +246,27 @@ export const JobsListPage: FC<{
                 id="kibanaManagementMLSection"
                 data-test-subj="mlPageStackManagementJobsList"
               >
-                {spacesEnabled && (
-                  <>
-                    <EuiButtonEmpty onClick={() => setShowSyncFlyout(true)}>
-                      {i18n.translate('xpack.ml.management.jobsList.syncFlyoutButton', {
-                        defaultMessage: 'Synchronize saved objects',
-                      })}
-                    </EuiButtonEmpty>
-                    {showSyncFlyout && <JobSpacesSyncFlyout onClose={onCloseSyncFlyout} />}
-                    <EuiSpacer size="s" />
-                  </>
-                )}
+                <EuiFlexGroup>
+                  <EuiFlexItem grow={false}>
+                    {spacesEnabled && (
+                      <>
+                        <EuiButtonEmpty onClick={() => setShowSyncFlyout(true)}>
+                          {i18n.translate('xpack.ml.management.jobsList.syncFlyoutButton', {
+                            defaultMessage: 'Synchronize saved objects',
+                          })}
+                        </EuiButtonEmpty>
+                        {showSyncFlyout && <JobSpacesSyncFlyout onClose={onCloseSyncFlyout} />}
+                        <EuiSpacer size="s" />
+                      </>
+                    )}
+                  </EuiFlexItem>
+                  <EuiFlexItem grow={false}>
+                    <ExportJobsFlyout isDisabled={false} currentTab={currentTabId} />
+                  </EuiFlexItem>
+                  <EuiFlexItem grow={false}>
+                    <ImportJobsFlyout refreshJobs={() => {}} isDisabled={false} />
```

Comment:
shall we make the `refreshJobs` prop optional?

Reply:
this is left over code, `refreshJobs` can be removed

Reply:
Updated in ba5a3fd